libpod/container.go

```diff
@@ -372,11 +372,11 @@ type ContainerConfig struct {
 	StopTimeout uint `json:"stopTimeout,omitempty"`
 	// Time container was created
 	CreatedTime time.Time `json:"createdTime"`
-	// NoCgroups indicates that the container will not create CGroups. It is
+	// CgroupsMode indicates that the container will not create CGroups. It is
```

Comment:
This looks like some over-enthusiastic search-and-replace...I suspect this should still read `NoCgroups`.

Reply:
indeed, good catch

Reply:
Aggressive rename